src/Command/MigrateCommand.php

```diff
@@ -223,10 +226,10 @@ private function executeCommand($name, $arguments)
         }
     }
 
-    private function guessReferenceType(string $assetCsvFilename): string
+    private function guessReferenceType(string $assetCsvFilename): ?string
     {
         try {
-            $this->io->writeln('The script will now guess if your assets are localizable, non localizable or both...');
+            $this->io->write('The command will parse your assets file... ');
```

Comment:
```suggestion
            $this->io->writeLn('The command will parse your assets file... ');
```

I found it confusing to have it in the same line

Reply:
I disagree... I found it confusing to have it in 2 different lines :p